pkg/operator/vitessbackup/storage.go

```diff
@@ -42,6 +42,8 @@ func StorageFlags(backupLocation *planetscalev2.VitessBackupLocation, clusterNam
 		return gcsBackupFlags(backupLocation.GCS, clusterName)
 	case backupLocation.S3 != nil:
 		return s3BackupFlags(backupLocation.S3, clusterName)
+	case backupLocation.Azblob != nil:
+		return azblobBackupFlags(backupLocation.Azblob, clusterName)
```

Comment:
Are these functions from something that hasn't been merged yet?

Reply:
Woops didn't add the file, yes comming in on this Commit now